job/server/src/main/java/alluxio/master/job/tracker/CmdJobTracker.java

```diff
@@ -143,9 +144,9 @@ public Status getCmdStatus(long jobControlId) throws JobDoesNotExistException {
 
     CmdInfo cmdInfo = mInfoMap.get(jobControlId);
 
-    if (cmdInfo.getCmdRunAttempt().isEmpty()) { // If no attempts created, throws an Exception
-      throw new JobDoesNotExistException(
-              ExceptionMessage.JOB_DEFINITION_DOES_NOT_EXIST.getMessage(jobControlId));
+    if (cmdInfo.getCmdRunAttempt().isEmpty()) { // If no attempts created,
```

Comment:
is there another situation that this would be empty other than files already loaded?

Reply:
Yes, if the loading directory has no files, then the job is empty